drv/gimlet-hf-server/src/main.rs

```diff
@@ -202,7 +202,7 @@ impl idl::InOrderHostFlashImpl for ServerImpl {
         self.check_muxed_to_sp()?;
         set_and_check_write_enable(&self.qspi)?;
         self.qspi.bulk_erase();
-        poll_for_write_complete(&self.qspi);
+        poll_for_write_complete(&self.qspi, Some(100));
```

Comment:
I didn't note this in the PR description, but I chose this sleep time based on the _fastest_ possible bulk erase time (similarly for sector erase). bulk_erase can take much, much longer (order of minutes), but spinning while sleeping for 100 ticks between each poll for minutes is probably okay?